framework/audio/engine/iaudioengine.h

```diff
@@ -0,0 +1,63 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#ifndef MU_AUDIO_IAUDIOENGINE_H
+#define MU_AUDIO_IAUDIOENGINE_H
+
+#include <string>
+#include <functional>
+
+#include "modularity/imoduleexport.h"
+
+#include "iaudiosource.h"
+
+namespace mu {
+namespace audio {
+namespace engine {
+class IAudioEngine : MODULE_EXPORT_INTERFACE
+{
+    INTERFACE_ID(IAudioEngine)
+
+public:
+    virtual ~IAudioEngine() = default;
+
+    using handle = unsigned int;
+    using time = float;
+
+    virtual bool init() = 0;
+    virtual void deinit() = 0;
+    virtual bool isInited() const = 0;
+
+    virtual float sampleRate() const = 0;
+
+    virtual handle play(IAudioSource* s, float volume = -1, float pan = 0, bool paused = false) = 0;
```

Comment:
bool paused = false

What is the purpose of this argument?

Reply:
https://github.com/musescore/MuseScore/pull/6332#discussion_r454924323